common/app/uiComponent/core/RenderingActor.scala

```diff
@@ -0,0 +1,23 @@
+package uiComponent.core
+
+import akka.actor.Actor
+import model.ApplicationContext
+import uiComponent.UIComponent
+import scala.util.Try
+
+case class Rendering(component: UIComponent, ac: ApplicationContext)
+
+case class RenderingException(error: String) extends RuntimeException(error)
+
+class RenderingActor extends Actor with JavascriptRendering {
+
+  override def javascriptFile: String = "ui.bundle.server.js"
```

Comment:
The RenderingActor doesn't know (or care) about what the javascript rendering returns. It might be a full html page, it might be a simple button, it doesn't matter as long as it is a string.
I am happy to rename `UIComponent` with something that encapsulates this is the representation of something that can be rendered, regardless of its complexity.
Writing this make me think that something like `Renderable` could be an acceptable `trait` name :D